src/main/java/com/jfrog/ide/idea/scan/data/applications/JFrogApplicationsConfig.java

```diff
@@ -1,19 +1,25 @@
 package com.jfrog.ide.idea.scan.data.applications;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
 import java.util.List;
 
+@Getter
+@NoArgsConstructor
 public class JFrogApplicationsConfig {
-   @JsonProperty("version")
-   private String version;
-   @JsonProperty("modules")
-   private List<ModuleConfig> modules;
+    @JsonProperty("version")
+    private String version;
+    @JsonProperty("modules")
+    private List<ModuleConfig> modules;
 
-   public String getVersion() {
-      return this.version;
-   }
+    public JFrogApplicationsConfig(boolean createEmptyModules) {
+        if (createEmptyModules) {
+            modules = new ArrayList<>();
+            modules.add(new ModuleConfig());
```

Comment:
Shouldn't we add a source root to the added module?
Like here: https://github.com/jfrog/jfrog-cli-core/blob/v2.44.1/xray/commands/audit/jas/common.go#L73

Reply:
It is done later in the flow, 
https://github.com/talarian1/jfrog-idea-plugin/blob/new-scanners-config/src/main/java/com/jfrog/ide/idea/scan/SourceCodeScannerManager.java#L206